Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "northstar-discord-rpc"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+rrplug = "0.1.0"
+discord-presence = { git = "https://github.com/jewlexx/discord-presence" } # TODO: move this to a version when the activity blocking is fixed
+
+[build-dependencies]
+windres = "0.2.2"
+
+[lib]
+crate-type = ["cdylib"]
```

Comment:
A few files are missing trailing newlines

![image](https://user-images.githubusercontent.com/40122905/201517645-dc8789b5-3c84-4e37-85fb-9a93ca289eba.png)

Trailing newlines are preferred as otherwise someone adding more code below will get the previous last line credited to them in `git  blame`, making it just slightly more annoying to find author of a certain line.

Reply:
Seems resolved with 31a3d714a042731fb79561fb2697b19cf12a4cf5